public_html/wp-content/plugins/wordcamp-organizer-reminders/wcor-reminder.php

```diff
@@ -194,11 +194,16 @@ protected function save_post_meta( $post, $new_meta ) {
 		}
 
 		if ( isset( $new_meta['wcor_send_when'] ) ) {
-			if ( in_array( $new_meta['wcor_send_when'], array( 'wcor_send_before', 'wcor_send_after', 'wcor_send_after_pending', 'wcor_send_after_and_no_report', 'wcor_send_trigger' ) ) ) {
+			if ( in_array( $new_meta['wcor_send_when'], array( 'wcor_send_before', 'wcor_send_after', 'wcor_send_after_pending', 'wcor_send_trigger' ) ) ) {
 				update_post_meta( $post->ID, 'wcor_send_when', $new_meta['wcor_send_when'] );
 			}
 		}
 
+		delete_post_meta( $post->ID, 'wcor_transparency_report' );
+		if ( isset( $new_meta['wcor_transparency_report'] ) ) {
+			update_post_meta( $post->ID, 'wcor_transparency_report', $new_meta['wcor_transparency_report'] );
```

Comment:
lets make sure to have some sanitization around this since it's just saving from $_POST

Reply:
4f0b6d1